HealthcareSystem/GraphicalEditor/Enumerations/MapObjectTypes.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphicalEditor.Enumerations
+{
+    public enum MapObjectTypes
+    {
+        EXAMINATION_ROOM,
+        OPERATION_ROOM,
+        WAITING_ROOM,
+        PARKING,
+        RESTAURANT,
+        WC,
+    }
```

Comment:
Ok, I'll do it. Our product owner hasn't mentioned hospitalization rooms when she was explaining that part, but I guess she has planned to leave for the next sprint.

Reply:
Sure, I assumed it was an accidental omission since all the other room types that exist on the backend are added already. I can't really see any harm in adding it right now though, and later it might be harder to do if more places that use it like the switch for colors appear?

Reply:
Yes, that's right. It will surely be needed for medicine and equipment manipulation later in the future.